scripts/test/run-tests.sh

```diff
@@ -160,5 +162,29 @@ else
                      || test_result=$?
   fi
 
+  if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "debian-qemux86_64" ]; then
+    wget --progress=dot:giga -N ${DEBIAN_IMAGE_URL} -P input/
+    convert_and_test "qemux86_64" \
+                     "release-1" \
+                     "input/Debian-11-x86-64.img.gz" \
+                     "--overlay tests/ssh-public-key-overlay" \
+                     "--config configs/debian-qemux86-64_config $EXTRA_CONFIG" \
+                     || test_result=$?
+
+    echo >&2 "----------------------------------------"
+    echo >&2 "Running the uncompressed test"
+    echo >&2 "----------------------------------------"
+    rm -rf deploy
+    gunzip --force "input/Debian-11-x86-64.img.gz"
+    run_convert "release-2" \
+                "input/Debian-11-x86-64.img" \
+                "--config configs/debian-qemux86-64_config $EXTRA_CONFIG" || test_result=$?
+    ret=0
+    test -f deploy/Debian-11-x86-64-qemux86_64-mender.img || ret=$?
+    assert "${ret}" "0" "Expected uncompressed file deploy/Debian-11-x86-64-qemux86_64-mender.img"
```

Comment:
This we don't need - it is enough with one uncompressed test for one image

Reply:
removed.